apollo-api-impl/pom.xml

```diff
@@ -86,6 +86,11 @@
 
         <!--test scope-->
         <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+         <dependency>
```

Comment:
a-ha, this is waht you meant you had a similar issue as I was having

Reply:
yep